pkg/destroy/ovirt/destroyer.go

```diff
@@ -33,27 +33,31 @@ func (uninstaller *ClusterUninstaller) Run() error {
 	}
 	defer con.Close()
 
-	err = uninstaller.removeVms(con)
-	uninstaller.Logger.Errorf("Removing VMs - error: %s", err)
-	err = uninstaller.removeTag(con)
-	uninstaller.Logger.Errorf("Removing Tag - error: %s", err)
-	err = uninstaller.removeTemplate(con)
-	uninstaller.Logger.Errorf("Removing Template - error: %s", err)
+	if err := uninstaller.removeVMs(con); err != nil {
```

Comment:
One thing that was happening before was that all removal was tried before exiting... And maybe we should keep that behavior

We could collect errors into a slice and then use https://godoc.org/k8s.io/apimachinery/pkg/util/errors#NewAggregate to return an aggregated error.

You can use errors.Wrap to wrap while collecting so that the result will have the correct context

Reply:
NVM, I missed that there was no return error :p which is weird, but same as previous code...